frontend/rpm/template.go

```diff
@@ -355,6 +411,27 @@ func (w *specWrapper) Install() fmt.Stringer {
 		cfg := w.Spec.Artifacts.ConfigFiles[c]
 		copyArtifact(`%{buildroot}/%{_sysconfdir}`, c, cfg)
 	}
+	serviceKeys := dalec.SortMapKeys(w.Spec.Artifacts.Services)
+	for _, p := range serviceKeys {
+		cfg := w.Spec.Artifacts.Services[p]
+		// must include '.service' suffix in name
```

Comment:
We should support more than `.service` files

Reply:
As in other file formats?

Reply:
There's other types of systemd units, e.g. `<foo>.mount` or `<foo>.socket`

Reply:
Ok. Do we want to add declarative options for these, to be extra explicit
```yaml
systemdUnits:
    mounts:
    sockets:
         src/foo.socket:
    services:
        src/foo.service:
    ...
```
We will need to handle sockets differently. I.e in the above example we would want to enable `foo.socket`, but not `foo.service` (assuming foo.service is supposed to be started once there are connections to foo.socket).  One complicating thing is that `.socket` units can override what service they start using the `Service=` option, which would make it harder for us to figure out what to do, unless the user tells us explicitly what should be enabled and disabled.

Reply:
If we're going to add sockets, perhaps we should flip the default to disabled and make the user write `enable: true` to be absolutely sure that we don't accidentally enable a service that is associated with, for example, a socket

Reply:
For our k8s builds, we need services. IF we need to handle things differently, would it make sense to handle the service case first and then worry about other units?

Reply:
I don't think we need to have special cases for different types of units.
They are all just files that get copied into some specific location.
As for different defaults and things I don't think its needed, each unit its its own thing.
A socket does reference a service, but its not on us to make sure the unit file is correct.
We are just starting the unit or not.

Reply:
Ok. I think the best path forward then is to just make users put `enable: true` as a subproperty of the unit config, otherwise it will default to disabled

Reply:
SGTM

Reply:
I've now implemented this.